filebeat/module/system/_meta/docs.asciidoc

```diff
@@ -13,6 +13,10 @@ include::../include/what-happens.asciidoc[]
 This module was tested with logs from OSes like Ubuntu 12.04, Centos 7, and
 macOS Sierra.
 
+This module requires the
```

Comment:
Looking at the code it seems this one only requires the geoip and no the user agent. But TBH I think we should make the two a general requirement for all modules to make it simpler for the user to always install just both.